shell/platform/android/test/io/flutter/embedding/engine/renderer/FlutterRendererTest.java

```diff
@@ -116,4 +119,31 @@ public void itStopsSurfaceTextureCallbackWhenDetached() {
     // Verify behavior under test.
     verify(fakeFlutterJNI, times(0)).markTextureFrameAvailable(eq(entry.id()));
   }
+
+  @Test
+  public void itRegistersExistingSurfaceTexture() {
+    // Setup the test.
+    FlutterRenderer flutterRenderer = new FlutterRenderer(fakeFlutterJNI);
+
+    fakeFlutterJNI.detachFromNativeAndReleaseResources();
+
+    SurfaceTexture surfaceTexture;
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
do we need this condition? `public SurfaceTexture (int texName)` doesn't seem deprecated

Reply:
Done